provider/filestore_test.go

```diff
@@ -1,314 +0,0 @@
-//go:build filestore
-// +build filestore
-
-// This Source Code Form is subject to the terms of the Mozilla Public
-// License, v. 2.0. If a copy of the MPL was not distributed with this
-// file, You can obtain one at https://mozilla.org/MPL/2.0/.
-
-package provider
-
-import (
-	"encoding/json"
-	"fmt"
-	"io/ioutil"
-	"os"
-	"testing"
-
-	"github.com/joho/godotenv"
-
-	pc "github.com/featureform/provider/provider_config"
-)
-
-func TestFileStore(t *testing.T) {
```

Comment:
this file doesn't even make sense for your stuff anymore. We probably will eventually need filestore tests but for now